changelog.json

```
@@ -1,5 +1,15 @@
 {
   "releases": {
+      "1.7.0-beta2": [
+        "[New] Rebase your current branch using a guided flow - #5953",
```

I'm not wedded to this but thoughts on being a bit more explanatory?
```suggestion
        "[New] Rebase another branch onto your current branch using a guided flow - #5953",
```

---

I like this change but it feels the wrong way around - the current branch is rebased onto another branch of your choosing. What about this?

```suggestion
        "[New] Rebase your current branch onto another branch using a guided flow - #5953",
```

---

Sounds great!